docs/src/getting_started.ipynb

```diff
@@ -8,7 +8,17 @@
    "source": [
     "# Getting Started with Temporian\n",
     "\n",
-    "[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/google/temporian/blob/last-release/docs/src/tutorials/getting_started.ipynb)"
+    "[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/google/temporian/blob/last-release/docs/src/tutorials/getting_started.ipynb)\n",
+    "\n",
+    "This guide will introduce you to the basics of Temporian, including:\n",
```

Comment:
Some of the reader will come directly to this page, skipping the readme. I would add a paragraph telling what Temporian is, and where is the main doc.